javascript/javascript1/week3/main.js

```diff
@@ -0,0 +1,135 @@
+// freeCodeCamp: @TatjanaOcedova
+
+//Item array removal
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+let nameToRemove = "Ahmad";
+
+// Write some code here
+for (let i = 0; i < names.length; i++) {
+  if (names[i] === nameToRemove) {
+    names.splice(i, 1);
+  }
+}
+// names.splice(1, 1); if I don't need to use nameToRemove
+// Code done
+
+console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+//When will we be there??
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+function infoTravelTime(speed, destinationDistance) {
+  let info = travelInformation.destinationDistance / travelInformation.speed;
+  let hours = Math.floor(info);
+  let minutes = Math.floor((info % 1) * 60);
+  return hours + " hours and " + minutes + " minutes";
+}
+
+const travelTime = infoTravelTime(travelInformation);
+console.log(travelTime); // 8 hours and 38 minutes
+
```

Comment:
just for fun (!), you could consider adding some corner cases checks inside the function- right now i can pass negative values for speed and negative distances, which is interesting, but perhaps a tad unrealistic :)